objects/SetupChecker/script.lua

```diff
@@ -1975,16 +1975,17 @@ function setupGainSpiritChoices(obj, choices)
         })
     end
     if count > 0 then
+        table.insert(buttons, {tag = "Text"})
```

Comment:
was this line intentionally added?

Reply:
Yes, it seemed the easiest way to offset the Reroll button away from the spirit choices some.

Reply:
can you add a comment to this mentioning this

Reply:
Done.